resolwe/flow/managers/listener/python_process_plugin.py

```diff
@@ -38,6 +38,9 @@
 # How many times to retry creating new object on slug collision error.
 OBJECT_CREATE_RETRIES = 10
 
+# How many objects to return in one chunk when iterating.
+MAX_CHUNK_SIZE = 10000
```

Comment:
It would make sense to use different values for different models.

Reply:
Indeed it would. The only thing that is model-specific in the listener is the expose plugin, which exposes a model.

The `iterate_chunk_size` is not a perfect fit, but I have put it there. Any other ideas?